pkgs/by-name/di/diffoscope/package.nix

```
@@ -257,6 +255,12 @@ python.pkgs.buildPythonApplication rec {
       ])
       # oggvideotools is broken on Darwin, please put it back when it will be fixed?
       ++ lib.optionals stdenv.hostPlatform.isLinux [ oggvideotools ]
+      # Causes an eval failure
+      # See https://github.com/NixOS/nixpkgs/issues/463873
+      ++ lib.optionals (stdenv.hostPlatform.system != "aarch64-linux") [
```

```suggestion
      ++ lib.optionals (!stdenv.hostPlatform.isLinux || !stdenv.hostPlatform.isAarch64) [
```

String comparisons with systems don't work consistently.

Also if you're adding this in, why change aapt?

---

Yeah maybe change this to the suggestion. Other than that it builds fine.